2025-01-02T13:02:04.244177+00:00 WARNING 'ansible_navigator.image_manager.puller._log_message' Image inspection failed, image assumed to be corrupted or missing
2025-01-02T13:02:04.244485+00:00 WARNING 'ansible_navigator.image_manager.puller._log_message' stdout: []

2025-01-02T13:02:04.244516+00:00 WARNING 'ansible_navigator.image_manager.puller._log_message' stderr: Error: inspecting object: ghcr.io/ansible/creator-ee:v0.22.0: image not known

2025-01-02T13:02:26.356548+00:00 CRITICAL 'ansible_navigator.actions.run_9577._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '763af41a-e988-411f-9a33-03ce4d52832a', 'counter': 1, 'stdout': "\x1b[0;31mERROR! couldn't resolve module/action 'cisco.ios.ios_facts'. This often indicates a misspelling, missing collection, or incorrect module path.\x1b[0m", 'start_line': 0, 'end_line': 1, 'runner_ident': '8f8b55b5-738c-4d87-8684-5b5e8009055f', 'created': '2025-01-02T13:02:26.355281+00:00'}
2025-01-02T13:02:26.356947+00:00 CRITICAL 'ansible_navigator.actions.run_9577._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': 'f6b429ae-1cb7-4a2e-915f-acbaa74728d9', 'counter': 2, 'stdout': '\x1b[0;31m\x1b[0m', 'start_line': 1, 'end_line': 2, 'runner_ident': '8f8b55b5-738c-4d87-8684-5b5e8009055f', 'created': '2025-01-02T13:02:26.355700+00:00'}
2025-01-02T13:02:26.357214+00:00 CRITICAL 'ansible_navigator.actions.run_9577._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '014aa7ee-0514-48d3-ae6a-d65b5331485f', 'counter': 3, 'stdout': "\x1b[0;31mThe error appears to be in '/home/ttweed98/ansible/aap_templates/facts.yml': line 8, column 7, but may\x1b[0m", 'start_line': 2, 'end_line': 3, 'runner_ident': '8f8b55b5-738c-4d87-8684-5b5e8009055f', 'created': '2025-01-02T13:02:26.355821+00:00'}
2025-01-02T13:02:26.357316+00:00 CRITICAL 'ansible_navigator.actions.run_9577._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': 'c478fa85-16b6-49c8-8b8f-ae65ec6d0a3c', 'counter': 4, 'stdout': '\x1b[0;31mbe elsewhere in the file depending on the exact syntax problem.\x1b[0m', 'start_line': 3, 'end_line': 4, 'runner_ident': '8f8b55b5-738c-4d87-8684-5b5e8009055f', 'created': '2025-01-02T13:02:26.355909+00:00'}
2025-01-02T13:02:26.357412+00:00 CRITICAL 'ansible_navigator.actions.run_9577._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '7d11d5d5-d04d-4045-b0d2-e6773eda910a', 'counter': 5, 'stdout': '\x1b[0;31m\x1b[0m', 'start_line': 4, 'end_line': 5, 'runner_ident': '8f8b55b5-738c-4d87-8684-5b5e8009055f', 'created': '2025-01-02T13:02:26.355981+00:00'}
2025-01-02T13:02:26.357512+00:00 CRITICAL 'ansible_navigator.actions.run_9577._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '4e860e8b-e7ba-4250-8fd5-cf56088a9d36', 'counter': 6, 'stdout': '\x1b[0;31mThe offending line appears to be:\x1b[0m', 'start_line': 5, 'end_line': 6, 'runner_ident': '8f8b55b5-738c-4d87-8684-5b5e8009055f', 'created': '2025-01-02T13:02:26.356045+00:00'}
2025-01-02T13:02:26.357607+00:00 CRITICAL 'ansible_navigator.actions.run_9577._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': 'fff25980-8cd4-4f1a-981e-9be4d343b8b1', 'counter': 7, 'stdout': '\x1b[0;31m\x1b[0m', 'start_line': 6, 'end_line': 7, 'runner_ident': '8f8b55b5-738c-4d87-8684-5b5e8009055f', 'created': '2025-01-02T13:02:26.356105+00:00'}
2025-01-02T13:02:26.357697+00:00 CRITICAL 'ansible_navigator.actions.run_9577._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': 'd762964a-28d9-4cbe-b662-af5fc53b2790', 'counter': 8, 'stdout': '\x1b[0;31m  tasks:\x1b[0m', 'start_line': 7, 'end_line': 8, 'runner_ident': '8f8b55b5-738c-4d87-8684-5b5e8009055f', 'created': '2025-01-02T13:02:26.356176+00:00'}
2025-01-02T13:02:26.357790+00:00 CRITICAL 'ansible_navigator.actions.run_9577._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '3a8a7bb0-0e4e-40ff-aacc-baa789ab4c0d', 'counter': 9, 'stdout': '\x1b[0;31m    - name: Gather facts\x1b[0m', 'start_line': 8, 'end_line': 9, 'runner_ident': '8f8b55b5-738c-4d87-8684-5b5e8009055f', 'created': '2025-01-02T13:02:26.356250+00:00'}
2025-01-02T13:02:26.357892+00:00 CRITICAL 'ansible_navigator.actions.run_9577._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '3b1c17e7-f7b9-4ffb-a11c-fcc565bd2d2a', 'counter': 10, 'stdout': '\x1b[0;31m      ^ here\x1b[0m', 'start_line': 9, 'end_line': 10, 'runner_ident': '8f8b55b5-738c-4d87-8684-5b5e8009055f', 'created': '2025-01-02T13:02:26.356321+00:00'}
2025-01-02T13:10:33.387584+00:00 CRITICAL 'ansible_navigator.actions.run_30b7._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '07e63139-e468-4aad-ba85-be59f7a61edd', 'counter': 1, 'stdout': "\x1b[0;31mERROR! couldn't resolve module/action 'cisco.ios.facts'. This often indicates a misspelling, missing collection, or incorrect module path.\x1b[0m", 'start_line': 0, 'end_line': 1, 'runner_ident': '13e7054a-55a2-4df9-b2a1-60c3c3abaa20', 'created': '2025-01-02T13:10:33.387159+00:00'}
2025-01-02T13:10:33.388396+00:00 CRITICAL 'ansible_navigator.actions.run_30b7._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': 'dbc4d0bf-05aa-4773-8b63-003531da2b48', 'counter': 2, 'stdout': '\x1b[0;31m\x1b[0m', 'start_line': 1, 'end_line': 2, 'runner_ident': '13e7054a-55a2-4df9-b2a1-60c3c3abaa20', 'created': '2025-01-02T13:10:33.387912+00:00'}
2025-01-02T13:10:33.388537+00:00 CRITICAL 'ansible_navigator.actions.run_30b7._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '255be355-f856-43d6-91a6-6eb94fffa9c9', 'counter': 3, 'stdout': "\x1b[0;31mThe error appears to be in '/home/ttweed98/ansible/aap_templates/facts.yml': line 8, column 7, but may\x1b[0m", 'start_line': 2, 'end_line': 3, 'runner_ident': '13e7054a-55a2-4df9-b2a1-60c3c3abaa20', 'created': '2025-01-02T13:10:33.388290+00:00'}
2025-01-02T13:10:33.389039+00:00 CRITICAL 'ansible_navigator.actions.run_30b7._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '4d01e082-1419-4ace-9a47-699c2eabfa9f', 'counter': 4, 'stdout': '\x1b[0;31mbe elsewhere in the file depending on the exact syntax problem.\x1b[0m', 'start_line': 3, 'end_line': 4, 'runner_ident': '13e7054a-55a2-4df9-b2a1-60c3c3abaa20', 'created': '2025-01-02T13:10:33.388663+00:00'}
2025-01-02T13:10:33.389218+00:00 CRITICAL 'ansible_navigator.actions.run_30b7._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '1f5a183b-912b-4542-9d3b-8954ee2e2e3a', 'counter': 5, 'stdout': '\x1b[0;31m\x1b[0m', 'start_line': 4, 'end_line': 5, 'runner_ident': '13e7054a-55a2-4df9-b2a1-60c3c3abaa20', 'created': '2025-01-02T13:10:33.388747+00:00'}
2025-01-02T13:10:33.389510+00:00 CRITICAL 'ansible_navigator.actions.run_30b7._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '0bcc081f-ae10-4b71-9746-26b072231079', 'counter': 6, 'stdout': '\x1b[0;31mThe offending line appears to be:\x1b[0m', 'start_line': 5, 'end_line': 6, 'runner_ident': '13e7054a-55a2-4df9-b2a1-60c3c3abaa20', 'created': '2025-01-02T13:10:33.388818+00:00'}
2025-01-02T13:10:33.389648+00:00 CRITICAL 'ansible_navigator.actions.run_30b7._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': 'fdd835fe-5458-4856-97a6-d25d279f16e3', 'counter': 7, 'stdout': '\x1b[0;31m\x1b[0m', 'start_line': 6, 'end_line': 7, 'runner_ident': '13e7054a-55a2-4df9-b2a1-60c3c3abaa20', 'created': '2025-01-02T13:10:33.388888+00:00'}
2025-01-02T13:10:33.389757+00:00 CRITICAL 'ansible_navigator.actions.run_30b7._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': 'ee7cc9f4-ff2e-4dd4-b25c-9512624ee299', 'counter': 8, 'stdout': '\x1b[0;31m  tasks:\x1b[0m', 'start_line': 7, 'end_line': 8, 'runner_ident': '13e7054a-55a2-4df9-b2a1-60c3c3abaa20', 'created': '2025-01-02T13:10:33.388960+00:00'}
2025-01-02T13:10:33.389867+00:00 CRITICAL 'ansible_navigator.actions.run_30b7._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '291e61b9-536d-4b5e-9312-bc5837f81abc', 'counter': 9, 'stdout': '\x1b[0;31m    - name: Gather facts\x1b[0m', 'start_line': 8, 'end_line': 9, 'runner_ident': '13e7054a-55a2-4df9-b2a1-60c3c3abaa20', 'created': '2025-01-02T13:10:33.389292+00:00'}
2025-01-02T13:10:33.389966+00:00 CRITICAL 'ansible_navigator.actions.run_30b7._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '6f81285f-85b7-46ac-98b8-849f693a9af7', 'counter': 10, 'stdout': '\x1b[0;31m      ^ here\x1b[0m', 'start_line': 9, 'end_line': 10, 'runner_ident': '13e7054a-55a2-4df9-b2a1-60c3c3abaa20', 'created': '2025-01-02T13:10:33.389398+00:00'}
2025-01-02T13:11:17.526753+00:00 CRITICAL 'ansible_navigator.actions.run_9b84._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': 'f219d9cc-9232-4f95-953a-8c814f294f5c', 'counter': 1, 'stdout': "\x1b[0;31mERROR! couldn't resolve module/action 'cisco.ios.ios_facts'. This often indicates a misspelling, missing collection, or incorrect module path.\x1b[0m", 'start_line': 0, 'end_line': 1, 'runner_ident': '815916bc-58fc-4bd9-a664-d899d3c0945e', 'created': '2025-01-02T13:11:17.526268+00:00'}
2025-01-02T13:11:17.528820+00:00 CRITICAL 'ansible_navigator.actions.run_9b84._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '2a51e81a-a39d-4ae6-8da6-fbb5acc8febd', 'counter': 2, 'stdout': '\x1b[0;31m\x1b[0m', 'start_line': 1, 'end_line': 2, 'runner_ident': '815916bc-58fc-4bd9-a664-d899d3c0945e', 'created': '2025-01-02T13:11:17.527155+00:00'}
2025-01-02T13:11:17.528932+00:00 CRITICAL 'ansible_navigator.actions.run_9b84._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '7c31c4e2-a01c-4ea0-af46-57a67864304e', 'counter': 3, 'stdout': "\x1b[0;31mThe error appears to be in '/home/ttweed98/ansible/aap_templates/facts.yml': line 8, column 7, but may\x1b[0m", 'start_line': 2, 'end_line': 3, 'runner_ident': '815916bc-58fc-4bd9-a664-d899d3c0945e', 'created': '2025-01-02T13:11:17.527265+00:00'}
2025-01-02T13:11:17.529011+00:00 CRITICAL 'ansible_navigator.actions.run_9b84._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '8da64df7-a670-4ef2-9271-008c8032ab5e', 'counter': 4, 'stdout': '\x1b[0;31mbe elsewhere in the file depending on the exact syntax problem.\x1b[0m', 'start_line': 3, 'end_line': 4, 'runner_ident': '815916bc-58fc-4bd9-a664-d899d3c0945e', 'created': '2025-01-02T13:11:17.527362+00:00'}
2025-01-02T13:11:17.529077+00:00 CRITICAL 'ansible_navigator.actions.run_9b84._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': 'ae3935dc-32c5-4943-9831-92e1ac3713bf', 'counter': 5, 'stdout': '\x1b[0;31m\x1b[0m', 'start_line': 4, 'end_line': 5, 'runner_ident': '815916bc-58fc-4bd9-a664-d899d3c0945e', 'created': '2025-01-02T13:11:17.527446+00:00'}
2025-01-02T13:11:17.529207+00:00 CRITICAL 'ansible_navigator.actions.run_9b84._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '35724b6e-a989-4321-bc8e-46d24b0a7373', 'counter': 6, 'stdout': '\x1b[0;31mThe offending line appears to be:\x1b[0m', 'start_line': 5, 'end_line': 6, 'runner_ident': '815916bc-58fc-4bd9-a664-d899d3c0945e', 'created': '2025-01-02T13:11:17.527628+00:00'}
2025-01-02T13:11:17.529279+00:00 CRITICAL 'ansible_navigator.actions.run_9b84._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': 'c57e445a-0fa3-4f7f-be63-a0249d7598ec', 'counter': 7, 'stdout': '\x1b[0;31m\x1b[0m', 'start_line': 6, 'end_line': 7, 'runner_ident': '815916bc-58fc-4bd9-a664-d899d3c0945e', 'created': '2025-01-02T13:11:17.527742+00:00'}
2025-01-02T13:11:17.529338+00:00 CRITICAL 'ansible_navigator.actions.run_9b84._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '3d29ef57-f33b-4641-8765-a635cd57d38a', 'counter': 8, 'stdout': '\x1b[0;31m  tasks:\x1b[0m', 'start_line': 7, 'end_line': 8, 'runner_ident': '815916bc-58fc-4bd9-a664-d899d3c0945e', 'created': '2025-01-02T13:11:17.527833+00:00'}
2025-01-02T13:11:17.529412+00:00 CRITICAL 'ansible_navigator.actions.run_9b84._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': 'da39fa5a-9a35-4226-9bb0-7280ea0542cb', 'counter': 9, 'stdout': '\x1b[0;31m    - name: Gather facts\x1b[0m', 'start_line': 8, 'end_line': 9, 'runner_ident': '815916bc-58fc-4bd9-a664-d899d3c0945e', 'created': '2025-01-02T13:11:17.527920+00:00'}
2025-01-02T13:11:17.529480+00:00 CRITICAL 'ansible_navigator.actions.run_9b84._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '307d910b-032c-453f-900b-579b4eb36a18', 'counter': 10, 'stdout': '\x1b[0;31m      ^ here\x1b[0m', 'start_line': 9, 'end_line': 10, 'runner_ident': '815916bc-58fc-4bd9-a664-d899d3c0945e', 'created': '2025-01-02T13:11:17.528062+00:00'}
2025-01-02T13:18:04.232931+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '0087352e-a2df-4ba7-b063-7db7497db0da', 'counter': 1, 'stdout': "\x1b[0;31mERROR! couldn't resolve module/action 'cisco.ios.facts'. This often indicates a misspelling, missing collection, or incorrect module path.\x1b[0m", 'start_line': 0, 'end_line': 1, 'runner_ident': '5942d187-eea2-43df-ad1c-aa9ea02c5783', 'created': '2025-01-02T13:18:04.231811+00:00'}
2025-01-02T13:18:04.243947+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '346c0813-5d96-4d6e-bf69-7fc2f0a0d503', 'counter': 2, 'stdout': '\x1b[0;31m\x1b[0m', 'start_line': 1, 'end_line': 2, 'runner_ident': '5942d187-eea2-43df-ad1c-aa9ea02c5783', 'created': '2025-01-02T13:18:04.233510+00:00'}
2025-01-02T13:18:04.244328+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '19e68d3a-b9e3-4111-ac0b-820ae732a111', 'counter': 3, 'stdout': "\x1b[0;31mThe error appears to be in '/home/ttweed98/ansible/aap_templates/facts.yml': line 7, column 7, but may\x1b[0m", 'start_line': 2, 'end_line': 3, 'runner_ident': '5942d187-eea2-43df-ad1c-aa9ea02c5783', 'created': '2025-01-02T13:18:04.233621+00:00'}
2025-01-02T13:18:04.244551+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '8c37cfc6-691d-4afe-a9c7-338b63450d0a', 'counter': 4, 'stdout': '\x1b[0;31mbe elsewhere in the file depending on the exact syntax problem.\x1b[0m', 'start_line': 3, 'end_line': 4, 'runner_ident': '5942d187-eea2-43df-ad1c-aa9ea02c5783', 'created': '2025-01-02T13:18:04.233691+00:00'}
2025-01-02T13:18:04.244729+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '9bf37776-4ddd-499e-90f4-52a2ccedf5e6', 'counter': 5, 'stdout': '\x1b[0;31m\x1b[0m', 'start_line': 4, 'end_line': 5, 'runner_ident': '5942d187-eea2-43df-ad1c-aa9ea02c5783', 'created': '2025-01-02T13:18:04.234491+00:00'}
2025-01-02T13:18:04.244948+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '4a166130-741c-4b12-b5af-fcf22e4f5f51', 'counter': 6, 'stdout': '\x1b[0;31mThe offending line appears to be:\x1b[0m', 'start_line': 5, 'end_line': 6, 'runner_ident': '5942d187-eea2-43df-ad1c-aa9ea02c5783', 'created': '2025-01-02T13:18:04.234566+00:00'}
2025-01-02T13:18:04.245166+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': 'ae778649-58ae-4af9-8cd1-ee8a216ac936', 'counter': 7, 'stdout': '\x1b[0;31m\x1b[0m', 'start_line': 6, 'end_line': 7, 'runner_ident': '5942d187-eea2-43df-ad1c-aa9ea02c5783', 'created': '2025-01-02T13:18:04.234623+00:00'}
2025-01-02T13:18:04.245369+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '98528233-f873-4597-aeae-4b0efd01171f', 'counter': 8, 'stdout': '\x1b[0;31m  tasks:\x1b[0m', 'start_line': 7, 'end_line': 8, 'runner_ident': '5942d187-eea2-43df-ad1c-aa9ea02c5783', 'created': '2025-01-02T13:18:04.234677+00:00'}
2025-01-02T13:18:04.245549+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '1452e109-26e2-4b75-b8bb-2f350230e11f', 'counter': 9, 'stdout': '\x1b[0;31m    - name: gather router facts\x1b[0m', 'start_line': 8, 'end_line': 9, 'runner_ident': '5942d187-eea2-43df-ad1c-aa9ea02c5783', 'created': '2025-01-02T13:18:04.234727+00:00'}
2025-01-02T13:18:04.245734+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '583995c7-8377-4f45-9186-9975c7f14b32', 'counter': 10, 'stdout': '\x1b[0;31m      ^ here\x1b[0m', 'start_line': 9, 'end_line': 10, 'runner_ident': '5942d187-eea2-43df-ad1c-aa9ea02c5783', 'created': '2025-01-02T13:18:04.234776+00:00'}
